### ISSUE TITLE
Download coco-pose dataset  fail.

### ISSUE THREAD

**Issue by yaoerqin** [ultralytics/ultralytics#21104]
### Search before asking

- [x] I have searched the Ultralytics YOLO [issues](https://github.com/ultralytics/ultralytics/issues) and found no similar bug report.

### Ultralytics YOLO Component

Train

### Bug

Hello there,

I tried to run the pose model by using coco-pose dataset. The zip files can be downloaded  correctly,  but it failed  suddendly. Is there any problem of the downloaded dataset?  The coco8-pose can be run correctly, but the full coco dataset was wrong.

![Image](https://github.com/user-attachments/assets/6290bd65-d355-4257-bc02-4194220d3edd)

![Image](https://github.com/user-attachments/assets/c305d161-947b-466f-aca3-47890a201eb0)

### Environment

Ultralytics 8.3.144  Python-3.11.7 torch-2.6.0+cu118 CUDA:0 (NVIDIA GeForce RTX 4090, 24564MiB)
Setup complete  (24 CPUs, 95.2 GB RAM, 1103.5/1858.0 GB disk)

OS                  Windows-10-10.0.22631-SP0
Environment         Windows
Python              3.11.7
Install             pip
Path                D:\Installation_Software\anaconda\Lib\site-packages\ultralytics
RAM                 95.21 GB
Disk                1103.5/1858.0 GB
CPU                 AMD Ryzen 9 7900X3D 12-Core Processor
CPU count           24
GPU                 NVIDIA GeForce RTX 4090, 24564MiB
GPU count           1
CUDA                11.8

numpy                1.26.4>=1.23.0
matplotlib           3.8.0>=3.3.0
opencv-python        4.10.0.84>=4.6.0
pillow               10.2.0>=7.1.2
pyyaml               6.0.1>=5.3.1
requests             2.31.0>=2.23.0
scipy                1.11.4>=1.4.1
torch                2.6.0+cu118>=1.8.0
torch                2.6.0+cu118!=2.4.0,>=1.8.0; sys_platform == "win32"
torchvision          0.21.0+cu118>=0.9.0
tqdm                 4.65.0>=4.64.0
psutil               5.9.0
py-cpuinfo           9.0.0
pandas               2.1.4>=1.1.4
ultralytics-thop     2.0.14>=2.0.0

### Minimal Reproducible Example

yolo pose train data=coco-pose.yaml model=yolo11n-pose.yaml epochs=1

### Are you willing to submit a PR?

- [ ] Yes I'd like to help by submitting a PR!

**Comment by UltralyticsAssistant** [comment]
👋 Hello @yaoerqin, thank you for your interest in Ultralytics 🚀! We recommend a visit to the [Docs](https://docs.ultralytics.com/) for new users where you can find many [Python](https://docs.ultralytics.com/usage/python/) and [CLI](https://docs.ultralytics.com/usage/cli/) usage examples and where many of the most common questions may already be answered.

Since this appears to be a 🐛 Bug Report, we appreciate you including a [minimum reproducible example](https://docs.ultralytics.com/help/minimum-reproducible-example/)—this helps us debug more effectively.

An Ultralytics engineer will review your report and assist you soon 🤖. In the meantime, please ensure you’re using the latest version of the package, as many issues are resolved in recent releases.

Join the Ultralytics community where it suits you best:
- For real-time chat, head to [Discord](https://discord.com/invite/ultralytics) 🎧
- For in-depth discussions, check out [Discourse](https://community.ultralytics.com/)
- Dive into threads on our [Subreddit](https://reddit.com/r/Ultralytics) to share knowledge with the community

## Upgrade

Upgrade to the latest `ultralytics` package including all [requirements](https://github.com/ultralytics/ultralytics/blob/main/pyproject.toml) in a [**Python>=3.8**](https://www.python.org/) environment with [**PyTorch>=1.8**](https://pytorch.org/get-started/locally/) to verify your issue is not already resolved in the latest version:

```bash
pip install -U ultralytics
```

## Environments

YOLO may be run in any of the following up-to-date verified environments (with all dependencies including [CUDA](https://developer.nvidia.com/cuda-zone)/[CUDNN](https://developer.nvidia.com/cudnn), [Python](https://www.python.org/) and [PyTorch](https://pytorch.org/) preinstalled):

- **Notebooks** with free GPU: <a href="https://console.paperspace.com/github/ultralytics/ultralytics"><img src="https://assets.paperspace.io/img/gradient-badge.svg" alt="Run on Gradient"/></a> <a href="https://colab.research.google.com/github/ultralytics/ultralytics/blob/main/examples/tutorial.ipynb"><img src="https://colab.research.google.com/assets/colab-badge.svg" alt="Open In Colab"></a> <a href="https://www.kaggle.com/models/ultralytics/yolo11"><img src="https://kaggle.com/static/images/open-in-kaggle.svg" alt="Open In Kaggle"></a>
- **Google Cloud** Deep Learning VM. See [GCP Quickstart Guide](https://docs.ultralytics.com/yolov5/environments/google_cloud_quickstart_tutorial/)
- **Amazon** Deep Learning AMI. See [AWS Quickstart Guide](https://docs.ultralytics.com/yolov5/environments/aws_quickstart_tutorial/)
- **Docker Image**. See [Docker Quickstart Guide](https://docs.ultralytics.com/yolov5/environments/docker_image_quickstart_tutorial/) <a href="https://hub.docker.com/r/ultralytics/ultralytics"><img src="https://img.shields.io/docker/pulls/ultralytics/ultralytics?logo=docker" alt="Docker Pulls"></a>

## Status

<a href="https://github.com/ultralytics/ultralytics/actions/workflows/ci.yml?query=event%3Aschedule"><img src="https://github.com/ultralytics/ultralytics/actions/workflows/ci.yml/badge.svg" alt="Ultralytics CI"></a>

If this badge is green, all [Ultralytics CI](https://github.com/ultralytics/ultralytics/actions/workflows/ci.yml?query=event%3Aschedule) tests are currently passing. CI tests verify correct operation of all YOLO [Modes](https://docs.ultralytics.com/modes/) and [Tasks](https://docs.ultralytics.com/tasks/) on macOS, Windows, and Ubuntu every 24 hours and on every commit.

**Comment by Y-T-G** [comment]
You can try deleting the folder and downloading again

**Comment by Y-T-G** [comment]
You can try deleting the folder and running again.

**Comment by yaoerqin** [comment]
I have delete all the folders and running again, but still failed.


Did you try running this dataset and success?




Huimin Yao
***@***.***



&nbsp;




------------------ 原始邮件 ------------------
发件人:                                                                                                                        "ultralytics/ultralytics"                                                                                    ***@***.***&gt;;
发送时间:&nbsp;2025年6月18日(星期三) 晚上9:28
***@***.***&gt;;
抄送:&nbsp;"Huimin ***@***.******@***.***&gt;;
主题:&nbsp;Re: [ultralytics/ultralytics] Download coco-pose dataset  fail. (Issue #21104)



Y-T-G left a comment (ultralytics/ultralytics#21104)
 
You can try deleting the folder and running again.

**Comment by Y-T-G** [comment]
Did you check if the image files exist in the zip file?

**Comment by yaoerqin** [comment]
> Did you check if the image files exist in the zip file?

The error means some of the images didn't exist in the folder, but  the folder is not empty. I tried to manually unzip the zip file and still meet  this  problem.

**Comment by yaoerqin** [comment]
> Did you check if the image files exist in the zip file?

Hi, did you try this coco  dataset before? How many hours did you take  for downloading it? I redownload it for almost 15 hours and download is still stuck and not making any progress.

**Comment by Y-T-G** [comment]
It should download fast. Maybe slow if you're in China.
